src/Cache/CacheOperationTypes/CacheOperationTypes.php

```diff
@@ -2,11 +2,25 @@
 namespace Momento\Cache\CacheOperationTypes;
 
 use Cache_client\_GetResponse;
+use Cache_client\_ListFetchResponse;
```

Comment:
Is the generated code not in a separate namespace like `Momento_protos`?

Reply:
No, it generates its own namespaces which are separate from ours. See `composer.json` for more details.